test/E2ETest/Microsoft.Test.E2E.AspNet.OData/BulkOperation/BulkInsertDataModel.cs

```diff
@@ -0,0 +1,181 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using Microsoft.AspNet.OData.Builder;
+using Purchasing;
```

Comment:
>Purchasing [](start = 6, length = 10)

what's "Purchasing"?

Reply:
and Sorting

---
In reply to: [604628854](https://github.com/OData/WebApi/pull/2455#discussion_r604628854) [](ancestors = 604628854)